add a github_widget shortcode

dacog opened the pull request:
Hi, 

I created a shortcode to show information for a GitHub repository and I thought it would be interesting for others as well, so here it is.

Please let me know if I have to change anything.

Best regards, 

Diego

Kwpolska commented:
This looks great, just one thing missing (from the README):

> Please make sure your plugin has a valid open-source license, and that it has a license comment. For short licenses ([MIT](https://choosealicense.com/licenses/mit/)/[BSD](https://choosealicense.com/licenses/bsd-2-clause/)), include the entire license text; for longer licenses (Apache, GPL/LGPL), use the short declarations ([Apache](https://www.apache.org/licenses/LICENSE-2.0#apply), [GPL](https://www.gnu.org/licenses/gpl-howto.html)). If you don’t know which license to use, we recommend MIT (as seen in Nikola core).

Please also fix the linting errors:

```
v8/github_widget/github_widget.py:2:1: F401 'nikola.utils.LOGGER' imported but unused
v8/github_widget/github_widget.py:64:9: F841 local variable 'latest_release' is assigned to but never used
v8/github_widget/github_widget.py:65:9: F841 local variable 'latest_commit' is assigned to but never used
```

dacog commented:
Hi @Kwpolska 

Thank you for your feedback. I thought I had read everything. I just added the License (MIT).

I also updated a part of the code to re-enable the last commit and last release args in the shortcode. It seems I had forgotten to push that part before.

The linting should also be fixed.

Would there still be anything missing?

Thanks!

Diego

Kwpolska commented:
Thanks for your contribution!
